Firebase/Utilities/Logger/GULLogger.m

```diff
@@ -0,0 +1,273 @@
+// Copyright 2017 Google
```

Comment:
I realize this was copied over, but does this count as new code and an updated copyright?